flash.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import print_function
```

Comment:
Can we move this file to be a module in the utils folder and then be invoke as a command line argument to build.py?

Btw, if some of this code came from a different project we need to keep the copyright and license on the file header.

Reply:
Should be fine. I'll add the license header.